apps/AEPSampleApp/App.tsx

```diff
@@ -93,14 +94,15 @@ const Drawer = createDrawerNavigator();
 
 export default function App() {
 
+  useEffect(() => {
```

Comment:
Can you make a comment to call out about initializing inside the `useEffect` hook while using a function based component or `componentDidMount` method while using class based component?